### PR TITLE
manager_user

### PR DESCRIPTION
**Pull request by datToc004**
## Related Tickets:
- [#40369](https://edu-redmine.sun-asterisk.vn/issues/40369)
-  [#40370](https://edu-redmine.sun-asterisk.vn/issues/40370)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by datToc004** [comment]
ready

File: app/Http/Controllers/Manager/UserController.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    {
+        dd(123);
+
+        return view('manager.user.index', compact('users'));
+    }
+
+    public function blockUser($id)
+    {
+        $user = User::findOrFail($id);
+        $user->status = 0;
```
Comment (manhtd-1083) [inline app/Http/Controllers/Manager/UserController.php:28]: config

File: app/Http/Controllers/Manager/UserController.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        $user->save();
+
+        return redirect()->route('users.index');
+    }
+
+    public function unBlockUser($id)
+    {
+        $user = User::findOrFail($id);
+        $user->status = 1;
```
Comment (manhtd-1083) [inline app/Http/Controllers/Manager/UserController.php:37]: đây nữa

**Comment by datToc004** [comment]
ready

**Review by manhtd-1083 — APPROVED** [review]

**Comment by manhtd-1083** [comment]
conflicted

**Comment by datToc004** [comment]
ready

**Review by manhtd-1083 — APPROVED** [review]

**Pull request merged**